File: src/main/java/com/example/interviewlandbackend/dto/response/SectionDto.java
package com.example.interviewlandbackend.dto.response;

public class SectionDto {

    private int id;

    private String sectionName;


    public SectionDto(int id, String sectionName) {
        this.id = id;
        this.sectionName = sectionName;
    }

    public int getId() {
        return id;
    }

    public String getSectionName() {
        return sectionName;
    }


}

File: src/main/java/com/example/interviewlandbackend/dto/response/SectionDto.kt
package com.example.interviewlandbackend.dto.response

import com.example.interviewlandbackend.model.Section

data class SectionDto @JvmOverloads constructor(

    val id: Int?,
    val sectionName: String

){
    companion object{

        @JvmStatic fun convert(section: Section) : SectionDto{
            return SectionDto(
                section.id,
                section.sectionName
            )
        }

    }


}